notebooks/text_classification/text_classification_introspective_rationale_explainer.ipynb

```diff
@@ -77,7 +79,8 @@
     "load_pretrained_model = False\n",
     "pretrained_model_path = \"../models/rnn.pth\"\n",
     "MODEL_SAVE_DIR = os.path.join(\"..\", \"models\")\n",
-    "model_prefix = \"sst2rnpmodel\""
+    "model_prefix = \"sst2rnpmodel\"\n",
+    "json_path = \"C:/interpret-text-contrib/introspective_rationale_config.json\""
```

Comment:
hmm this looks like a hard-coded path to c: drive, was that intended?  maybe you can make it relative path at least?

Reply:
thanks for catching that. Its an unused param so removing it!